assets/css/_nav.scss

```diff
@@ -1,3 +1,4 @@
+@use "sass:map";
```

Comment:
Just curious: Why is `sass:map` used here but not in any other document?

Reply:
Ah yes, good catch! It is an unnecessary leftover of trying to use the map incorrectly. Originally I was using `map.get` instead of `map-get` (frustratingly based on the [sass documentation](https://sass-lang.com/documentation/values/maps#look-up-a-value)) which didn't work, so this was an attempt at a fix that was ultimately unnecessary.